### PR TITLE
Release Notes: Use head_commit to get the merge commit message

### PR DESCRIPTION
**Pull request by karinathomasbbc**
**Overall change:**
When merging to latest, a GitHub action creates release notes - see https://github.com/bbc/simorgh/releases
However, the message is incorrect - it is using the first commit message instead of the merge commit message

**Code changes:**
- Use `head_commit` instead of `commits[0]` 

---

- [ ] I have assigned myself to this PR and the corresponding issues
- [ ] I have added the `cross-team` label to this PR if it requires visibility across World Service teams
- [ ] I have assigned this PR to the Simorgh project

**Testing:**

- [ ] Automated (jest and/or cypress) tests added (for new features) or updated (for existing features)
- [ ] If necessary, I have run the local E2E non-smoke tests relevant to my changes (`CYPRESS_APP_ENV=local CYPRESS_SMOKE=false npm run test:e2e:interactive`)
- [ ] This PR requires manual testing

### CODE REVIEW

File: .github/workflows/simorgh-create-github-release.yml [hunk 1]
```diff
@@ -29,6 +29,6 @@ jobs:
           tag_name: ${{ steps.release_tag.outputs.new_tag }}
           release_name: Release ${{ steps.release_tag.outputs.new_tag }}
           body: |
-            ${{ github.event.commits[0].message }}
-          draft: false
+            ${{ github.event.head_commit.message }}
```
Comment (karinathomasbbc) [inline .github/workflows/simorgh-create-github-release.yml:32]: `head_commit` is the commit when you merge the branch to latest e.g. in https://github.com/bbc/simorgh/commit/3b7dfcb0614b9cc88b992c004094d92fc7443206, the message is:
```
Merge pull request #8298 from bbc/github-release-notes

Use merge commit as release notes for release created by GitHub actions
```

**Review by RichardPK — APPROVED** [review]
A _perfect_ message for this doesn't exist, but this is probably our best option and significantly better than using the last commit message (how many 'update snapshots' releases must we have, ha). If Github auto-links to the PR # that would be fab.

**Comment by karinathomasbbc** [comment]
> A _perfect_ message for this doesn't exist, but this is probably our best option and significantly better than using the last commit message (how many 'update snapshots' releases must we have, ha). If Github auto-links to the PR # that would be fab.

Yeah, the message is far from perfect, but so much better than it used to be - https://github.com/bbc/simorgh/releases/tag/1.332.0 🙈

**Review by HarveyPeachey — APPROVED** [review]
A lot better than the old commit messages

**Comment by karinathomasbbc** [comment]
> A lot better than the old commit messages

If it works! 🤞 Here goes :)

**Pull request merged**